### ISSUE TITLE
model 'portal' can't use 

### ISSUE THREAD

**Issue by lonelyleaves** [odoo/odoo#64646]
Impacted versions:14.0

Steps to reproduce:  open app list; install model 'portal' and 'website', and open website

Current behavior:  can't open website
odoo log:    Valueerror: element '< XPath expr = / / * [hasclass ('O')_ footer_ copyright_ Name ')] ">' not found in parent view

Video/Screenshot link (optional):

![image](https://user-images.githubusercontent.com/16489920/104833080-15acfe00-58d1-11eb-90b5-012c4438ea8a.png)

**Comment by pedrobaeza** [comment]
Hi, thanks for reporting. Are you able to reproduce the problem in runbot infrastructure (http://runbot.odoo.com/runbot - click on the blue door icon of the version row - admin/admin)?

If so, please explain with all the possible detailed steps to get to the error. Not having them, it's very difficult to hunt down the problem. Please don't be shy of adding details, and even better if you provide some screenshots or videos.

If not, this can be due to a configuration change, a custom/community module you have deployed in your DB or an outdated code. Please report back if it's the case.

For keeping a clean house in this issue tracker, if you don't answer in several days, this will be closed assuming it was one of the stated things.

**Comment by lonelyleaves** [comment]
yes  ，I have reproduce this problem in “ https://5803817-saas-14-1.runbot46.odoo.com/web/login”


step
First, change the default language to Chinese,

Then install the module: portal

Final cancellation

**Comment by Yenthe666** [comment]
Perhaps yet another chinese translation issue @pedrobaeza?

**Comment by pedrobaeza** [comment]
Yes, it seems so. Please see https://github.com/odoo/odoo/wiki/Translations for knowing how to translate and look for the wrong translation.
